WordOfTheDay.Repository/WordsRepository.cs

```diff
@@ -64,23 +60,21 @@ public async Task PostWord(Word word)
         }
         public Task<bool> IsAlreadyExist(Word word)
         {
-            var time = DateTime.Today.ToUniversalTime();
-
             var exist = _context.Words
-                .AnyAsync(w => w.Email == word.Email && w.AddTime > time);
+                .LaterThan()
+                .AnyAsync(w => w.Email == word.Email);
 
             return exist;
         }
         public async Task<WordCount> UserWord(string email)
         {
-            var time = DateTime.Today.ToUniversalTime();
-
             var word = await _context.Words
-                .Where(word => word.AddTime > time)
+                .LaterThan()
                 .SingleOrDefaultAsync(w=>w.Email==email);
```

Comment:
You can use .ByEmail() extension here, also

Reply:
done